### PR TITLE
feat: Add basic evaluators for string experiment outputs

### LINKED ISSUE #3484: [experiments] contains_keyword

### LINKED ISSUE #3520: [experiments] isJSON code evaluator

### PR DESCRIPTION
**Pull request by anticorrelator**
resolves #3484 
resolves #3520

### CODE REVIEW

File: src/phoenix/datasets/experiments.py [hunk 1]
```diff
@@ -118,6 +119,48 @@ def _phoenix_client() -> httpx.Client:
...
+        self, input: JSONSerializable, reference: JSONSerializable, output: JSONSerializable
+    ) -> EvaluationProtocol:
+        assert isinstance(output, str), "Experiment run ooutput must be a string"
+        try:
+            json.loads(output)
+            json_parsable = True
+        except json.JSONDecodeError:
+            json_parsable = False
+
+        return EvaluationProtocol(
```
Comment (RogerHYang) [inline src/phoenix/datasets/experiments.py:135]: i don't think we should be using `Protocol` as instantiations

**Review by RogerHYang — APPROVED** [review]

File: src/phoenix/datasets/experiments.py [hunk 2]
```diff
@@ -118,6 +119,48 @@ def _phoenix_client() -> httpx.Client:
     return client
 
 
+class JSONParsable(ExperimentEvaluator):
+    annotator_kind = "CODE"
+
+    def __call__(
+        self, input: JSONSerializable, reference: JSONSerializable, output: JSONSerializable
+    ) -> EvaluationProtocol:
+        assert isinstance(output, str), "Experiment run ooutput must be a string"
```
Comment (axiomofjoy) [inline src/phoenix/datasets/experiments.py:128]:
```suggestion
        assert isinstance(output, str), "Experiment run output must be a string"
```

File: src/phoenix/datasets/experiments.py [hunk 3]
```diff
@@ -118,6 +119,48 @@ def _phoenix_client() -> httpx.Client:
     return client
 
 
+class JSONParsable(ExperimentEvaluator):
+    annotator_kind = "CODE"
```
Comment (axiomofjoy) [inline src/phoenix/datasets/experiments.py:123]: Might make sense to make this an enum.

File: src/phoenix/datasets/experiments.py [hunk 4]
```diff
@@ -118,6 +119,48 @@ def _phoenix_client() -> httpx.Client:
...
+            score=float(json_parsable),
+            explanation=None,
+            metadata={},
+        )
+
+
+class ContainsKeyword(ExperimentEvaluator):
+    annotator_kind = "CODE"
+
+    def __init__(self, contains: str):
```
Comment (axiomofjoy) [inline src/phoenix/datasets/experiments.py:145]:
```suggestion
    def __init__(self, substring: str):
```

`contains` sounds like a boolean.

**Review by axiomofjoy — APPROVED** [review]

File: src/phoenix/datasets/experiments.py (hunk 2 above)
Comment (anticorrelator) [reply src/phoenix/datasets/experiments.py]: thanks! fixed

File: src/phoenix/datasets/experiments.py (hunk 4 above)
Comment (anticorrelator) [reply src/phoenix/datasets/experiments.py]: good point, going with "keyword" to match the object verbiage

**Pull request merged**